composer.json

```diff
@@ -17,26 +17,26 @@
     ],
     "require": {
         "php": ">=7.0",
-        "doctrine/orm": "2.5.*|2.6.*",
-        "illuminate/auth": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/console": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/container": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/contracts": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/pagination": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/routing": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/validation": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/view": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "symfony/serializer": "^2.7|~3.0|~4.0"
+        "doctrine/orm": "2.5.*|2.6.*|2.7.*",
+        "illuminate/auth": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0.0",
```

Comment:
Can we drop support for 5.5? This version supports PHP 7.0 which is EOL now.

It would also allow us to merge #388.

_I would prefer to drop all except 6.0. As Laravel 6.0 requires 7.2 and 7.1 is EOL later this year._

Reply:
If a maintainer agrees should the required PHP version also be updated.